app/models/project.rb

```diff
@@ -240,15 +240,14 @@ def storage_capacity_xml
   end
 
   def storage_capacity(session_id:)
-    requested_capacity = storage_capacity_xml
```

Comment:
Can you also clean up the storage_capacity_xml function? https://github.com/pulibrary/tigerdata-app/blob/26bf465c12a856282cbd10065a0ade1b3a11d70d/app/models/project.rb#L237